cmake/FindCUDA/FindCUDA.cmake

```diff
@@ -788,21 +836,31 @@ if(CUDA_VERSION VERSION_EQUAL "3.0")
     )
 endif()
 
-if(CUDA_USE_STATIC_CUDA_RUNTIME AND NOT CUDA_VERSION VERSION_LESS "5.5")
```

Comment:
I think this will start linking pytorch against libcudart_static.a rather than the current libcudart.so. I am not entirely sure how that plays with conda which ships with libcudart.so as part of `cudatoolkit`

Reply:
This change came from:

```
commit aab1f6e984e81f732ec2dd5c33ff8f7bf604a0f2
Author: Stephen Sorley <Stephen.Sorley@jhuapl.edu>
Date:   Thu Aug 11 15:20:17 2016 -0400

    FindCUDA: Restore default behavior of CUDA_USE_STATIC_CUDA_RUNTIME
    
    Fix bug introduced by commit 7229ae72 (FindCUDA: Refactor Android(Tegra)
    support, 2016-04-19).  `CUDA_USE_STATIC_CUDA_RUNTIME` should be enabled
    by default if `cudart_static` is available, and silently disabled if
    it is not.
```

https://gitlab.kitware.com/cmake/cmake/commit/aab1f6e984e81f732ec2dd5c33ff8f7bf604a0f2

Based on the description, and also reading the code, it looks that if we set `-DCUDA_USE_STATIC_CUDA_RUNTIME=0`, we should recover the old behavior. cmake defaults to this ON (in fact, even our copy of cmake says as much). @soumith does this make sense?